config/types/passwd.go

```diff
@@ -65,9 +65,6 @@ type Group struct {
 func init() {
 	register2_0(func(in Config, ast validate.AstNode, out ignTypes.Config, platform string) (ignTypes.Config, report.Report, validate.AstNode) {
 		for _, user := range in.Passwd.Users {
-			if user.Name != "core" && user.Create == nil {
```

Comment:
Did this make its way into the wrong commit?

Reply:
Just moved it into a more appropriate commit